tests/tests_certificate.yml

```diff
@@ -0,0 +1,65 @@
+---
+- name: test - setup
+  hosts: all
+  tasks:
+    - name: Download current linux-system-roles.certificate
+      git:
+        repo: https://github.com/linux-system-roles/certificate/
+        dest: roles/linux-system-roles.certificate
+        version: master
+        depth: 1
+      delegate_to: 127.0.0.1
+      become: false
+
+- name: Install cockpit
+  hosts: all
+  vars:
+    cockpit_packages: minimal
+  roles:
+    - linux-system-roles.cockpit
+
+- name: Generate self-signed certmonger certificate
+  hosts: all
+  tasks:
+    # Fixed in cockpit 255 (https://github.com/cockpit-project/cockpit/commit/6ec4881856e)
+    - name: Allow certmonger to write into Cockpit's certificate directory
+      file:
+        path: /etc/cockpit/ws-certs.d/
+        state: directory
+        setype: cert_t
+
+    # has to be done dynamically, as the first step checks it out
+    - include_role:
+        name: linux-system-roles.certificate
+      vars:
+        certificate_requests:
+          - name: /etc/cockpit/ws-certs.d/monger-cockpit
+            dns: ['www.example.com', 'localhost']
+            ca: self-sign
+            group: cockpit-ws
+
+    # FIXME
+    - name: HACK - group option does not make the files group-readable  # noqa 302 deprecated-command-syntax
+      command: chmod g+r /etc/cockpit/ws-certs.d/monger-cockpit.crt /etc/cockpit/ws-certs.d/monger-cockpit.key
```

Comment:
so you can't use the `file` module with `mode` here???  https://docs.ansible.com/ansible/2.9/modules/file_module.html#parameter-mode